runner/ansible/vars/azure_env.yml

```diff
@@ -17,3 +17,13 @@ expected:
   "1.3.4": "3"
   "1.3.5": "60"
   "1.3.6": "120"
+  "1.4.1.monitor_interval": "3600"
+  "1.4.1.monitor_timeout": "120"
+  "1.4.1.pcmk_action_limit": "3"
+  "1.4.1.pcmk_monitor_retries": "4"
+  "1.4.1.pcmk_reboot_timeout": "900"
+  "1.4.1.power_timeout": "240"
+  "1.4.2": "correct OR fence_azure_arm not configured"
+  "1.5.1": "pass"
+  "1.5.2": "blocked"
```

Comment:
In fact, we don't need any of these `1.4.2`, `1.5.1`, `1.5.2` and `1.5.3` results here